ci-templates/native-build-steps.yaml

```diff
@@ -30,9 +31,17 @@ jobs:
     - ${{ if eq(parameters.keycloak, 'true') }}:
         - script: docker run --rm --publish 8180:8080 --name build-keycloak  -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=admin -d quay.io/keycloak/keycloak
           displayName: 'start keycloak'
+    - ${{ if eq(parameters.mysql, 'true') }}:
+        - bash: |
+            sudo apt-get update &&
+            sudo apt-get -y install mysql-server libmysqlclient-dev &&
+            mysql -e "create user 'hibernate_orm_test'@'localhost' identified by 'hibernate_orm_test';" -uroot -proot &&
+            mysql -e "create database hibernate_orm_test; grant all on hibernate_orm_test.* to 'hibernate_orm_test'@'localhost';" -uroot -proot
+          displayName: 'install mysql'
```

Comment:
So, AFAICS, we only test MySQL in the native case?

Reply:
Yes, I was hesitant on adding JVM test since it supposed to work OOTB and did not want to put too much strain on the CI for that  :-).
Maybe we should add one for the reasons you stated above. I'll do it.